source/solvers/gls_navier_stokes.cc

```diff
@@ -1269,6 +1278,33 @@ GLSNavierStokesSolver<dim>::set_initial_condition_fd(
           this->pcout << "*********************************" << std::endl;
         }
     }
+  else if (initial_condition_type ==
+           Parameters::InitialConditionType::average_velocity_profile)
+    {
+      this->pcout << "*********************************************************"
+                  << std::endl;
+      this->pcout
+        << " Initial condition using average velocity from checkpoint "
+        << std::endl;
+      this->pcout << "*********************************************************"
+                  << std::endl;
```

Comment:
There is a function in utilities called announce string
https://chaos-polymtl.github.io/lethe/doxygen/utilities_8h.html#a56a967052620929f4db03cee351d6890

You can use that to announce strings